constants.py

```diff
@@ -6,4 +6,4 @@
 QOS_MAYBE = 0
 
 
-FORMAT_TIME = '%Y-%m-%dT%H:%M:%SZ'
+FORMAT_TIME = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
What was the purpose of changing this? Just wondering 🤔

Reply:
`black` did, not me